app/src/main/res/raw/clash.yaml

```diff
@@ -0,0 +1,510 @@
+#port: 7890
+socks-port: 1080
+udp: true
+allow-lan: false
+mode: Rule
+log-level: silent
+#experimental:
+#  ignore-resolve-fail: true
+
+dns:
+  enable: true
+  listen: 0.0.0.0:10053
+  enhanced-mode: fake-ip
+  fake-ip-range: 255.0.128.1/20 # 255.0.128.1  - 255.0.143.254
+  nameserver:
+    - 114.114.114.114
+
+
+
+Proxy:
+- { name: "trojan", type: socks5, server: "127.0.0.1", port: 1081, udp: true}
+
+Proxy Group:
+- { name: "Proxy", type: select, proxies: ["trojan"] }
+
+
+Rule:
```

Comment:
Who maintains the rules? The gfwlist is bad enough even though it's regularly updated. If you ever want to keep a basic level of usability here the maintenance of rules alone is going to turn into a logistics nightmare.

That's not even accounting for the 8MB bloat of Clash being introduced here.

Reply:
1. The domain rules serve as a supplement of GEOIP rules.
1. It is a temporary solution to build in the rules. We should provide a config editor later.